lib/rubocop/cop/style/access_modifier_declarations.rb

```diff
@@ -67,6 +67,28 @@ module Style
       #     private :bar, :baz
       #
       #   end
+      #
+      # @example AllowModifiersOnAttrs: true (default)
```

Comment:
I think those examples should be extended to feature something else besides attributes, so people would understand the idea is to augment whatever the primary enforced style of the cop is.

Reply:
I would like to confirm that my understanding is correct, should I add method definitions like this?

``` ruby
# good
class Foo

  public attr_reader :bar
  protected attr_writer :baz
  private attr_accessor :qux
  private attr :quux

  def public_method; end

  private

  def private_method; end

end
```

Reply:
Yep.